src/input.js

```diff
@@ -1,50 +1,82 @@
 export const create = () => {
-  if (localStorage.getItem('todo') == null) {
-    localStorage.setItem('todo', JSON.stringify([]));
+  if (localStorage.getItem('lists') == null) {
+    localStorage.setItem('lists', JSON.stringify([]));
   }
-  return JSON.parse(localStorage.getItem('todo'));
+  return JSON.parse(localStorage.getItem('lists'));
 };
 
 const input = document.querySelector('.main');
 export const mydata = () => {
   input.addEventListener('keypress', (e) => {
+    const totaltasks = create();
     if (e.key === 'Enter') {
-      const totaltasks = create();
+      const l = totaltasks.length + 1;
       const task = {
-        check: true,
-        des: document.querySelector('.main').value,
-        index: totaltasks.length,
+        description: document.querySelector('.main').value,
+        completed: false,
+        index: totaltasks.length + 1,
```

Comment:
- [ ] Good that you have the tasks index set to the value of the new array length :+1: But please if I delete a task,  Your localStorage does not update all remaining items' indexes, so they represent the current list order as you can see below :point_down:  and are unique so please make sure you fix that :pray: 
  
  [Screencast from 07-21-2023 05:15:55 PM.webm](https://github.com/MayPyone/to-do-list/assets/92303891/e1fab279-f3e2-438d-9da8-14b0b01b7837)